support-frontend/assets/components/headers/header/header.scss

```diff
@@ -213,3 +212,100 @@ $header-links-min-width: gu-span(12);
     }
   }
 }
+
+.component-header--display-checkout {
+  .component-header__wrapper {
+    max-width: 50rem;
+
+    @include mq($from: wide) {
```

Comment:
I think this needs to be @include mq($from: leftCol) or what ever the variable name is for one down from wide